components/social/org.wso2.carbon.social.core/src/main/java/org/wso2/carbon/social/core/ActivityBrowser.java

```diff
@@ -37,11 +38,14 @@ JsonObject getSocialObject(String targetId, String order, int offset,
 
 	JsonObject getTopComments(String targetId, int likes) throws SocialActivityException;
 
+	JsonObject getUserComment(String userId, String targetId) throws SocialActivityException;
+
 	boolean isUserlikedActivity(String userId, int targetId, int like) throws SocialActivityException;
 
+	boolean isReviewed(String targetId, String userId) throws SocialActivityException;
+
 	JsonObject pollNewestComments(String targetId, int timestamp) throws SocialActivityException;
 
 	JsonObject getPopularAssets(String type, String tenantId, int limit,
 			int offset) throws SocialActivityException;
-
 }
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/980f09b60e89e3fe333c53c442a74375c315cb3d)